### ISSUE TITLE
[BUG] Shortcuts send 2 or 3 seconds later

### ISSUE THREAD

**Issue by chris25197** [Waboodoo/HTTP-Shortcuts#233]
Hello,

i have a few seconds ( 2 or 3) lag on Version 2.6.0 when i start a Shortcut (Get or Post request). 
Example: i open my door with io-Broker and Shelly 2.0. This trigger a door buzzer. 

Testet HTTP-Shortcuts on Version 2.5.0 or older, trigger the door buzzer instantly.

**Context (please complete the following information):**
 - Android OS Version: 10 and 11 
 - App Version 2.6.0

Thanks for help or Bugfixing.

By Chris.

**Comment by Waboodoo** [comment]
Hi Cris,

Could it be that you're using mDNS for this, i.e., the hostname of your URL ends in ".local"?

**Comment by chris25197** [comment]
Hi,

no, I use a local DNS Server. And yes, the url  ends with ".local".

Is that a problem in Version 2.6.0? 

Am 9. August 2021 10:57:14 MESZ schrieb Roland Meyer ***@***.***>:

**Comment by Waboodoo** [comment]
I have made an attempt of implementing mDNS/Zeroconf support into the app, but unfortunately Android is so inherently bad at it that I had to give up. There is now a workaround in place since 2.6.0, which seems to be causing the issue that you are now experiencing. It affects any shortcut which uses a ".local" URL. I will adjust it for the next version to mitigate this delay, and will look for a better solution. Sorry for the inconvenience.
